### ISSUE TITLE
`[clarity-repl]` Expose method to construct JSON abi from contract source

### ISSUE THREAD

**Issue by aulneau** [hirosystems/clarinet#913]
I'm working on an index of clarity contracts, and I want the ability to convert a string representation (standard .clar contents as string) of a contract into a JSON ABI, similar to the results from the stacks-blockchain-api/node. 

There are lots of use cases I have planned for it, but generally it's so I'm able to construct different views of a contract or find similarities between many.

**Comment by hugocaillard** [comment]
Hey @aulneau,
Is think the idea would be to expose `build_contract_interface` in the repl,
Something like
```rs
/* components/clarity-repl/src/lib.rs */

/* ... */

#[cfg(feature = "wasm")]
#[wasm_bindgen]
pub fn get_contract_interface(name: &str) {
    let mut session = unsafe {
        match WASM_GLOBAL_CONTEXT.session.take() {
            Some(session) => session,
            None => return (),
        }
    };
    build_contract_interface(session. /* get analysis ... */);
    ()
}
```

I could try to dig a bit deeper over the weekend or early next week

**Comment by smcclellan** [comment]
@aulneau  Is this still needed?

**Comment by hugocaillard** [comment]
Just moved this issue in progress, it will be available in the v1.0.0 of the clarinet-sdk #1065 

For context, @aulneau a new clarinet-sdk will be released next week on npm (release candidate 1). So if you're still interested, let's talk 🙌
